doc/production/pull-changes.md

```diff
@@ -25,8 +25,7 @@ Now that you have the code, run these commands:
     bundle install
     npm install
     rake db:migrate
-    rake assets:precompile # includes `npm run build`
-    bin/build-apidocs.sh
+    rake assets:precompile # includes `npm run build` and `bin/build-apidocs.sh`
```

Comment:
Looks like its switching to `rails` so should it say 'rails assets:precompile' now?